DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_bum_traffic_port_with_rif.py

```diff
@@ -0,0 +1,397 @@
+            "type" :"raw"
+        },
+        "IPv4_EIGRP": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "srcMac": srcMac,
+            "dstMac": "01:00:5E:00:00:0A",
+            "frameSize": 96,
+            "protocol": "ip",
+            "type" :"raw"
+        },
+        "IPv4_DHCPServerRelay": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "srcMac": srcMac,
+            "dstMac": "01:00:5E:00:00:0C",
+            "ipproto": "udp",
+            "srcPort": str(68),
+            "dstPort": str(67),
+            "frameSize": 96,
+            "protocol": "ip",
+            "type" :"raw"
+        },
+        "IPv4_VRRP": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "srcMac": srcMac,
+            "dstMac": "01:00:5E:00:00:12",
+            # "packet.ipv4.protocol": 112,
+            "frameSize": 96,
+            "protocol": "ip",
+            "type" :"raw"
+        },
+        "IPv4_IGMP" : {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "frameSize": 96,
+            "srcMac": srcMac,
+            "dstMac": "01:00:5E:00:00:16",
+            "protocol": "ip",
+            "type" :"raw"
+        },
+        "IPV4_BGP": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "ipproto": "tcp",
+            "srcPort": str(179),
+            "dstPort": str(179),
+            "srcMac": srcMac,
+            "dstMac": self_mac,
+            "frameSize": 96,
+            "protocol": "ip",
+            "type" :"raw"
+        }
+    }
+
+    await tgen_utils_setup_streams(tgen_dev, config_file_name=None, streams=streams)
+
+    tcpdump = asyncio.create_task(tb_device_tcpdump(dent_dev, "swp1", "-n", count_only=False, timeout=5, dump=True))
+
+    await tgen_utils_start_traffic(tgen_dev)
+    await asyncio.sleep(traffic_duration)
+    await tgen_utils_stop_traffic(tgen_dev)
+
+   # check the traffic stats
+    stats = await tgen_utils_get_traffic_stats(tgen_dev, "Traffic Item Statistics")
+    for row in stats.Rows:
+        assert tgen_utils_get_loss(row) == 100.000, \
+        f"Verify that traffic from {row['Tx Port']} to {row['Rx Port']} not forwarded.\n{out}"
+
+    await tcpdump
+    print(("TCPDUMP: packets=%s" % tcpdump.result()))
```

Comment:
use .format() or f-string:
```suggestion
    print(f"TCPDUMP: packets={tcpdump.result()}")
```